### DISCUSSION TITLE
Перенос расширения на Safari (Mac)

### DISCUSSION THREAD

**Discussion by moschidale** [anticensority/runet-censorship-bypass#206] [Ideas]
Подскажите, пожалуйста, планируется ли создание (перенос) этого расширения на Safari (Mac)? Если это возможно, конечно. Спасибо!

**Comment by ilyaigpetrov** [comment]
1) У меня нет Мака, чтобы написать порт или новое расширение.
2) Также расширение несколько устарело, его бы заново переписать, но я не гарантирую, что найду на это время.
3) Методы блокировок не стоят не месте -- может оказаться, что расширение заблокруют так, что даже свои прокси не помогут.

**Comment by moschidale** [comment]
Понял, большое спасибо за ответ
Дело в том, что на Safari я также не нашел и других подобных расширений, работающих с прокси, поэтому были сомнения, что подобные прокси-расширения вовсе там не поддерживаются, не работают и так далее; думал проблема в этом

**Comment by ilyaigpetrov** [comment]
Я тоже не в кусре их политик, но нашёл вот такое:
https://apps.apple.com/ru/app/%D0%B1%D0%B5%D1%81%D0%BF%D0%BB%D0%B0%D1%82%D0%BD%D1%8B%D0%B9-vpn-%D0%BE%D1%82-planet-vpn/id1410235921
.
